### ISSUE TITLE
WARNING: NODE_ENV value of 'production' did not match any deployment config file names.

### ISSUE THREAD

**Issue by nodecentral** [koalazak/rest980#73]
Hi,

I’ve just tried to install the docker instance and it seems to get stuck after that `NODE_ENV ` error is presented. Extract of console below..

````
[~] # docker run -e BLID=6801234560205690 -e PASSWORD=:1:1663852991:Notmyrealone0lP -e ROBOT_IP=10.10.10.50 -e KEEP_ALIVE=no koalazak/rest980
Unable to find image 'koalazak/rest980:latest' locally
latest: Pulling from koalazak/rest980
c5e155d5a1d1: Already exists 
221d80d00ae9: Already exists 
4250b3117dca: Already exists 
3b7ca19181b2: Pull complete 
425d7b2a5bcc: Pull complete 
69df12c70287: Pull complete 
ea2f5386a42d: Pull complete 
d421d2b3c5eb: Pull complete 
43ce3a2f0321: Pull complete 
d52690b14519: Pull complete 
5cb2b27b211b: Pull complete 
f207f5f82b01: Pull complete 
Digest: sha256:60ad4af0d55d8ef8f7ff01706againnottheactualonec36e01ac14db55cc6323d67018f0
Status: Downloaded newer image for koalazak/rest980:latest

> rest980@2.1.1 start /usr/src/app
> node ./bin/www

WARNING: NODE_ENV value of 'production' did not match any deployment config file names.
WARNING: See https://github.com/lorenwest/node-config/wiki/Strict-Mode
````

Any ideas ?

**Comment by jeremywillans** [comment]
This was fixed in #61, but a new image has not been deployed to Docker Hub. 

@koalazak - perhaps you can update the image at some stage?

**Comment by koalazak** [comment]
it is just a warning. I will update the docker image
